compiler/plugins/input/Torch/torch-iree/PluginRegistration.cpp

```diff
@@ -72,6 +75,28 @@ struct TorchSession
     typeMnemonics.insert("tm_tensor");
     typeMnemonics.insert("torch");
   }
+
+  std::string detectCustomInputOperations(ModuleOp &module) override {
+    auto *ctx = module.getContext();
+
+    const Dialect *torchDialect = ctx->getLoadedDialect("torch");
+    const Dialect *torchConversionDialect = ctx->getLoadedDialect("torch_c");
+    const Dialect *tmTensorDialect = ctx->getLoadedDialect("tm_tensor");
+
+    std::string typeMnemonic = "";
```

Comment:
Could torchDialect etc be null here?

Reply:
I hope not 👀 maybe I should check

Reply:
It could be if you use the API functions out of order. I'm not sure if it's worth checking for that vs letting the code just crash/error/fall-through.